blazor/common/how-to/publishAOTapplication.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
move content under https://blazor.syncfusion.com/documentation/common/deployment.

H2 heading

Ahead-of-time (AOT) compilation

-> provide known issue github link using @ref

Reply:
Moved content to deployment section with Heading Ahead-of-time (AOT) compilation